tests/Linter/ProcessLinterTest.php

```diff
@@ -62,10 +62,12 @@ public function testPrepareCommandOnPhpOnLinux($executable, $file, $expected)
      */
     public function testPrepareCommandOnPhpOnWindows($executable, $file, $expected)
     {
-        $this->assertSame(
-            $expected,
-            AccessibleObject::create(new ProcessLinter($executable))->prepareProcess($file)->getCommandLine()
-        );
+        if (self::hackyPleaseRemoveIsHostUsingWindows()) {
```

Comment:
we do have:
`testPrepareCommandOnPhpOnLinux`
`testPrepareCommandOnPhpOnWindows`

instead of this `hacky...`, add `testPrepareCommandOnPhpOnMac` and fix os detection